src/main/java/webserver/RequestHandler.java

```diff
@@ -1,55 +1,39 @@
 package webserver;
 
-import java.io.DataOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.net.Socket;
 
+import http.EnumHttpStatus;
+import http.request.HttpRequest;
+import http.request.HttpRequestExecutor;
+import http.response.HttpResponse;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class RequestHandler implements Runnable {
-    private static final Logger logger = LoggerFactory.getLogger(RequestHandler.class);
 
-    private Socket connection;
+  private static final Logger logger = LoggerFactory.getLogger(RequestHandler.class);
 
-    public RequestHandler(Socket connectionSocket) {
-        this.connection = connectionSocket;
-    }
+  private Socket connection;
 
-    public void run() {
-        logger.debug("New Client Connect! Connected IP : {}, Port : {}", connection.getInetAddress(),
-                connection.getPort());
-
-        try (InputStream in = connection.getInputStream(); OutputStream out = connection.getOutputStream()) {
-            // TODO 사용자 요청에 대한 처리는 이 곳에 구현하면 된다.
-            DataOutputStream dos = new DataOutputStream(out);
-            byte[] body = "Hello World".getBytes();
-            response200Header(dos, body.length);
-            responseBody(dos, body);
-        } catch (IOException e) {
-            logger.error(e.getMessage());
-        }
-    }
+  public RequestHandler(Socket connectionSocket) {
+    this.connection = connectionSocket;
+  }
 
-    private void response200Header(DataOutputStream dos, int lengthOfBodyContent) {
-        try {
-            dos.writeBytes("HTTP/1.1 200 OK \r\n");
-            dos.writeBytes("Content-Type: text/html;charset=utf-8\r\n");
-            dos.writeBytes("Content-Length: " + lengthOfBodyContent + "\r\n");
-            dos.writeBytes("\r\n");
-        } catch (IOException e) {
-            logger.error(e.getMessage());
-        }
-    }
+  public void run() {
+    logger.debug("New Client Connect! Connected IP : {}, Port : {}", connection.getInetAddress(),
+        connection.getPort());
 
-    private void responseBody(DataOutputStream dos, byte[] body) {
-        try {
-            dos.write(body, 0, body.length);
-            dos.flush();
-        } catch (IOException e) {
-            logger.error(e.getMessage());
-        }
+    try (InputStream in = connection.getInputStream(); OutputStream out = connection.getOutputStream()) {
```

Comment:
이렇게 줄을 분리하면 가독성 확보에 도움이 됩니다
```suggestion
    try (InputStream in = connection.getInputStream(); 
         OutputStream out = connection.getOutputStream()) {
```

Reply:
https://github.com/kakao-2022-summer/be-w234-was/pull/55/commits/5d7090016e020759aa5f020f3996f646ac8ca2c0

반영했습니다